GroupDocs.Editor.MVC/Products/Editor/Controllers/EditorApiController.cs

```diff
@@ -354,114 +392,179 @@ private dynamic GetSaveFormat(string saveFilePath)
                 default:
                     format = WordProcessingFormats.Docx;
                     break;
-
             }
+
             return format;
         }
 
-        private dynamic GetSaveOptions(string saveFilePath)
+        private ILoadOptions GetLoadOptions(string guid)
         {
-            string extension = Path.GetExtension(saveFilePath).Replace(".", "");
+            string extension = Path.GetExtension(guid).Replace(".", "");
             extension = CultureInfo.CurrentCulture.TextInfo.ToTitleCase(extension);
+
             if (extension.Equals("Txt"))
             {
                 extension = "Text";
             }
-            dynamic options = null;
-            foreach (var item in Enum.GetNames(typeof(WordProcessingFormats)))
+
+            ILoadOptions options = null;
+
+            foreach (var item in typeof(WordProcessingFormats).GetFields())
             {
-                if (item.Equals("Auto"))
+                if (item.Name.Equals("Auto"))
                 {
                     continue;
                 }
-                if (item.Equals(extension))
+
+                if (item.Name.Equals(extension))
                 {
-                    options = new WordProcessingSaveOptions();
+                    options = new WordProcessingLoadOptions();
                     break;
                 }
             }
+
             if (options == null)
             {
-                options = new SpreadsheetSaveOptions();
+                options = new SpreadsheetLoadOptions();
             }
+
             return options;
         }
 
-        private static List<string> PrepareFormats()
+        private IEditOptions GetEditOptions(string guid)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetEditOptions' a static method.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5172828)